### ISSUE TITLE
[Improvement]: An Appimage for Linux

### ISSUE THREAD

**Issue by RafaelLinux** [omega-gg/MotionBox#30]
Unfortunately, I got too many error when trying to launch under openSUSE. Really I would like to give a try, but nowadays, when we have AppImage format, I think you shoud build better an AppImage.

Errors:

`./start.sh 
Starting MotionBox...
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libtorrent-rasterbar.so.9)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libQt5Widgets.so.5)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libQt5Quick.so.5)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libQt5Gui.so.5)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libQt5Qml.so.5)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libQt5XmlPatterns.so.5)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libQt5Core.so.5)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libQt5Core.so.5)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libpng16.so.16)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libicui18n.so.66)
/home/falele/bin/MotionBox-1.6.0-12/MotionBox: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /home/falele/bin/MotionBox-1.6.0-12/libicuuc.so.66)
`

Thank you

**Comment by 3unjee** [comment]
You're absolutely right I have to look into this, but that's not my complete priority at the moment.

So I'm open to contributions for this :)
